### DISCUSSION TITLE
[v5] query max function is undefined

### DISCUSSION THREAD

**Discussion by b23618** [adonisjs/core#2658] [Help]
`const userlastId = Database.rawQuery("SELECT MAX(id) as lastId from users");
console.log(userlastId[0].lastId)
    return userlastId[0].lastId;`

message error **Cannot read property 'lastId' of undefined**

**Comment by RomainLanz** [comment]
Hey @menphurk! 👋 

Calling the `Database` is an asynchronous call, you need to `await` it.
📚 https://docs.adonisjs.com/reference/database/main#rawquery

Be more polite when asking a question, and do some debugging next time. :)
